src/components/InterruptedVideosGallery.tsx

```diff
@@ -24,13 +24,19 @@ const InterruptedVideosGallery: React.FC<RouteComponentProps> = () => {
     updateWatchedVideos('REMOVED', id)
   }
 
+  const onVideoNotFound = (id: string) => {
+    console.warn(`interrupted video not found, removing id: ${id}`)
```

Comment:
```suggestion
    console.warn(`Interrupted video not found, removing id: ${id}`)
```
Dunno, looks just better with capital letter.